_posts/2019-03-17-three-years-of-react-native.markdown

```diff
@@ -0,0 +1,174 @@
+front-end at Artsy. It wasn't surprising when the aspect with the most company focus became much larger. What
+creates tricky constraints are that the few people with the interest and skills to work on our React Native
+infrastructure also happen to have the skills to work on the web problems too.
+
+Those web problems tend to be a lot more valuable to the business.
+
+There's some work that can be done to benefit both, but in the last year it has been hard to prioritise
+iOS-specific platform work. For example, it took almost a year to get around to upgrading our version of React
+Native. This is a pretty risky place to be for a platform which we care about.
+
+At the end of 2018, we came to the conclusion that this was something we wanted to work to improve. So, we
+specifically brought back an explicit iOS-specific culture to Artsy - when we split our front-end practice into web
+and [iOS][fe-ios].
+
+Maybe this is a small admission of failure to the idea of a purely de-silo'd team, but realistically while product
+work across the two front-ends teams is consistent - the platform concerns just aren't.
+
+## Keeping up with the Jobs'
+
+As individual developers, it's much less important for us to keep up to date with the latest Apple developer news.
+WWDCs and iOS releases just kinda happen, and we keep moving independently. We still support iOS 9, and have very
+little incentive to bump it - new APIs from Apple just aren't that interesting anymore.
+
+This is a side-effect to the openness of the JavaScript toolchain, and our ability to contribute and fix a lot of
+our own problems at JavaScript level instead of needing to dive deeper into the native side to work on problems.
+
+Without a focused iOS team, it's not easy to pitch for iOS specific projects. This means that features like moving
+to Apple's new app store review API or using iOS features which don't exist on web gets talked about in meetings,
+but never started. This lack of a focused team makes it really hard to implement fixes to bad App Store reviews
+
+That said, in the last year we did manage to ship a pretty hefty [ARKit feature][arkit] - which a positive example
+of an iOS-specific feature which is both forward-thinking tech and a super great fit for Artsy's product. React
+Native basically played no part in that.
+
+## Community Disconnection
+
+There are maybe four communities at play if you're doing React Native:
+
+- Node (Babel/TypeScript/Storybooks/VSCode)
+- React (Hooks/Relay/Styled Components)
+- React Native (Mostly people creating cross-platform tools)
+- Native (Obj-C/Swift/CocoaPods/Fastlane etc)
+
+It's hard to keep on-top of any one community, and it's very hard to keep on track of four. Realistically, if you
+want to be writing apps at the level of quality we do - you need to though.
+
+Being involved in so many communities, as a part of owning [the Omakase][oma] stack means that it's hard to engage
```

Comment:
I’m unsure what the “as a part of owning the Omakase stack” section is meant to say here. I feel like the sentence needs reworking.